garrysmod/lua/postprocess/dof.lua

```diff
@@ -13,12 +13,16 @@ local NUM_DOF_NODES = 16
 
 function DOF_Kill()
 
-	for k, v in pairs( DOF_Ents ) do
+	for i = #DOF_Ents, 1, -1 do

```

Comment:
Looping through in reverse means we can safely set `DOF_Ents[i] = nil` without the cost of reordering any elements, and without needing to do another loop or call `table.Empty`

Reply:
Is there any reason to bother with this over just doing `DOF_Ents = {}` at the end of the function?